draft-ietf-masque-h3-datagram.md

```diff
@@ -623,68 +314,30 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
-# The Sec-Use-Datagram-Contexts HTTP Header {#context-hdr}
+# The Capsule-Protocol Header
 
-Endpoints indicate their support for datagram contexts by sending the
-Sec-Use-Datagram-Contexts header with a value of ?1. If the header is missing
-or has a value different from ?1, that indicates that its sender does not wish
-to use datagram contexts. Endpoints that wish to use datagram contexts SHALL
-send the Sec-Use-Datagram-Contexts header with a value of ?1 on requests and
-responses that use the capsule protocol.
+Definitions of new HTTP Methods or of new HTTP Upgrade Tokens that use the
+Capsule Protocol MAY use the Capsule-Protocol header field to simplify
+intermediary processing.
 
-"Sec-Use-Datagram-Contexts" is an Item Structured Header {{!RFC8941}}. Its
-value MUST be a Boolean, its ABNF is:
+"Capsule-Protocol" is an Item Structured Header {{!RFC8941}}. Its value MUST be
+a Boolean, its ABNF is:
 
 ~~~
-Sec-Use-Datagram-Contexts = sf-boolean
+Capsule-Protocol = sf-boolean
```

Comment:
I am inclined to make this `Sec-`, because I don't see a scenario in which JavaScript would need to send it, and in general I believe that new request headers should be defined as `Sec-` unless they're both safe and useful for JavaScript applications.

Reply:
It sounds like we have folks who prefer with `Sec-` and folks who prefer without. Let's discuss in person.

Reply:
In today's design team meeting, we reached rough consensus to not use the `Sec-` prefix. We still have [an issue](https://github.com/ietf-wg-masque/draft-ietf-masque-connect-udp/issues/68) to track adding text to Security Considerations, so I'm marking this comment thread as resolved.